artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/PageTimedWriter.java

```diff
@@ -118,15 +120,15 @@ public void incrementTask() {
       pendingTasksUpdater.incrementAndGet(this);
    }
 
-   public void addTask(OperationContext context,
+   public int addTask(OperationContext context,
                                     PagedMessage message,
                                     Transaction tx,
-                                    RouteContextList listCtx) {
+                                    RouteContextList listCtx, boolean useFlowControl) {
 
       logger.trace("Adding paged message {} to paged writer", message);
 
       int credits = Math.min(message.getEncodeSize() + PageReadWriter.SIZE_RECORD, maxCredits);
-      writeCredits.acquireUninterruptibly(credits);
+      //writeCredits.acquireUninterruptibly(credits);
```

Comment:
Commented out code is usually confusing for future maintenance, recommend removing this line

Reply:
oops.. thanks... I forgot to remove it...

I kept adding it back and forth during my testing (to reproduce the issue) and I forgot to remove it.. thanks